app/src/main/java/com/framgia/englishconversation/screen/imagedetail/ImageDetailViewModel.java

```diff
@@ -22,14 +24,27 @@
     private FragmentManager mManager;
     private CommentFragment mFragment;
     private Context mContext;
-    private OnMediaModelItemTouchListener<List<MediaModel>> mListener =
+    private OnMediaModelItemTouchListener mListener =
             new OnMediaModelItemTouchListener<List<MediaModel>>() {
                 @Override
                 public void onTouchListener(List<MediaModel> items, int postion) {
                     mContext.startActivity(
                             SelectedImageDetailActivity.getIntent(mContext, items, postion));
                 }
             };
+    private OnTimelineItemTouchListener mTouchListener =
+            new OnTimelineItemTouchListener<TimelineModel>() {
+                @Override
+                public void onItemTimelineClick(TimelineModel item) {
+                    //TODO
+                }
+
+                @Override
+                public void onItemUserNameClick(TimelineModel item) {
+                    mContext.startActivity(
```

Comment:
ERROR: 'method def' child have incorrect indentation level 20, expected level should be one of the following: 12, 16.